### PR TITLE
Glimpse: Generify media update in MediaViewer

### PR DESCRIPTION
**Pull request by luca020400**
Change-Id: I3eae0c0b11b58a202f7075a0ae995e2defc64954

### CODE REVIEW

File: app/src/main/java/org/lineageos/glimpse/thumbnail/MediaViewerAdapter.kt [hunk 1]
```diff
@@ -40,7 +41,10 @@ class MediaViewerAdapter(
     )
 
     override fun onBindViewHolder(holder: MediaViewHolder, position: Int) {
-        getMediaFromMediaStore(position)?.let { holder.bind(it, position) }
+        getMediaFromMediaStore(position)?.let {
+            holder.bind(it, position)
+            mediaUpdateCallback(it, position)
```
Comment (SebaUbuntu) [inline app/src/main/java/org/lineageos/glimpse/thumbnail/MediaViewerAdapter.kt]: Why is this called here? onBindViewHolder is called before the view is shown

Comment (SebaUbuntu) [reply app/src/main/java/org/lineageos/glimpse/thumbnail/MediaViewerAdapter.kt]: Ah you check the position before doing stuff

Comment (luca020400) [reply app/src/main/java/org/lineageos/glimpse/thumbnail/MediaViewerAdapter.kt]: Right, on viewAttached is better

Comment (luca020400) [reply app/src/main/java/org/lineageos/glimpse/thumbnail/MediaViewerAdapter.kt]: NAK, breaks callback expectations

**Pull request closed without merging**